crypto/merkle/rfc6962_test.go

```diff
@@ -0,0 +1,99 @@
+package merkle
+
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// These tests were taken from https://github.com/google/trillian/blob/master/merkle/rfc6962/rfc6962.go,
+// and consequently fall under the above license.
+
+import (
+	"bytes"
+	"encoding/hex"
+	"testing"
+
+	"github.com/tendermint/tendermint/crypto/tmhash"
+)
+
+func TestRFC6962Hasher(t *testing.T) {
+	_, leafHashTrail := trailsFromByteSlices([][]byte{[]byte("L123456")})
+	leafHash := leafHashTrail.Hash
+	_, leafHashTrail = trailsFromByteSlices([][]byte{[]byte{}})
+	emptyLeafHash := leafHashTrail.Hash
+
+	for _, tc := range []struct {
+		desc string
+		got  []byte
+		want string
+	}{
```

Comment:
Any reason the 1st test vector is left out?
https://github.com/google/trillian/blob/4179aaf7ae95e21a606b8d79e8d8c3397c826fa6/merkle/rfc6962/rfc6962_test.go#L41-L46

Reply:
We don't support making a merkle tree of nothing (As far as I'm aware)